### DISCUSSION TITLE
How do I configure apollo-angular to operate entirely client side with no server?

### DISCUSSION THREAD

**Discussion by rDr4g0n** [the-guild-org/apollo-angular#1727] [Q&A]
I am prototyping a schema purely client side with no server (the server is under development separately). All of the apollo-angular docs assume you will set up an `HttpLink` with a valid `uri`, but I have no `uri` to point to. Every field in my schema is decorated with `@client`, so there is no need to go across the wire to any server.

How can I configure my client to be 100% encapsulated within the UI?

Additionally, if this is possible, it should be mentioned in the [Development and Testing](https://apollo-angular.com/docs/development-and-testing/client-schema-mocking) doc.

**Comment by Urigo** [comment]
that's a good thing to add to the docs.
It is also not apollo-angular specific but apollo-client, maybe you can open an issue there, start with a simple example and get help from the community to create an example?
